docs/services/vision/detection.md

```diff
@@ -82,34 +144,90 @@ The optional **saturation_cutoff_pct** and **value_cutoff_pct** attributes speci
 
 {{% /alert %}}
 
-### `tflite_detector`
+Click **Save config** and head to the **Components** tab.
+Proceed to [Add a camera component and a "transform" model](#add-a-camera-component-and-a-transform-model).
+
+## Configure a `tflite_detector`
 
 A machine learning detector that draws bounding boxes according to the specified tensorflow-lite model file available on the robot’s hard drive.
 
+### Create the ML Model Service
+
+Navigate to the [robot page on the Viam app](https://app.viam.com/robots), then create an ML Model Service for the classifier model:
+
+{{< tabs >}}
+{{% tab name="Builder" %}}
+Click on the robot you wish to add the classifier to.
+Select the **config** tab, and click on **Services**.
+
+Scroll to the **Create Service** section:
+
+1. Select `ml_model` as the **Type**.
```

Comment:
the type is just "mlmodel" rather than "ml_model"

It's a bit confusing, because the model in the vision service is called "ml_model" ...

Reply:
fixed in the code so everything is "mlmodel"